### PR TITLE
Update to 1.25.2

### PR DESCRIPTION
**Pull request by ninadingole**
resolves #295

### CODE REVIEW

File: scripts/update-patch.sh [hunk 1]
```diff
@@ -9,8 +9,8 @@ pushd "$SCRIPTS_DIR/../playwright"
 SCRIPTS_DIR="$(dirname "$0")"
 echo "Creating patch..."
 git add .
-git diff --full-index playwright-build^1..playwright-build > ../patches/main.patch
+git diff --full-index 396487f..main > ../patches/main.patch
```
Comment (mxschmitt) [inline scripts/update-patch.sh:12]: this looks not how it should be.

File: run.go [hunk 2]
```diff
@@ -13,9 +13,13 @@ import (
 	"os/exec"
```
Comment (mxschmitt) [inline run.go:13]: the diff main.patch is too big. did you create it with the scripts?

Comment (ninadingole) [reply run.go:13]: no, I think I created it wrong. It shouldn't be this big

File: scripts/update-patch.sh (hunk 1 above)
Comment (ninadingole) [reply scripts/update-patch.sh:12]: yeah this part I didn't understand that's why I asked you how to create the patch and then use it

Comment (ninadingole) [reply scripts/update-patch.sh:12]: I fixed the tests all and the test works with `1.25.2` version, I am not able to get the types/interface generation part

**Comment by mxschmitt** [comment]
@ninadingole be aware, I did something and force-pushed, looks not that bad. I don't remember how the patching stuff was working, was a while ago the last time I touched it, might require some more investigation, optimization and docs.

Some tests are failing it looks like, might require some more work.

**Comment by ninadingole** [comment]
> @ninadingole be aware, I did something and force-pushed, looks not that bad. I don't remember how the patching stuff was working, was a while ago the last time I touched it, might require some more investigation, optimization and docs.
> 
> Some tests are failing it looks like, might require some more work.

Thanks @mxschmitt it looks like the CI failed for some domain issue and not because of the test problem. I locally ran all the tests and they passed without any issue. Maybe retrying CI again will work

**Comment by mxschmitt** [comment]
Merging for now, we can follow up with making the contribution guidelines better, cleaning up code etc.

**Pull request merged**